src/jit/gentree.cpp

```diff
@@ -15580,9 +15580,19 @@ bool GenTree::canBeContained() const
 {
     assert(IsLIR());
 
+    if (gtHasReg())
+    {
+        return false;
+    }
+
     // It is not possible for nodes that do not produce values or that are not containable values
     // to be contained.
-    return (OperKind() & (GTK_NOVALUE | GTK_NOCONTAIN)) == 0;
```

Comment:
I would leave this as-is as a matter of taste.

Reply:
Because now we return false `if gtHasReg()`  I want to leave it in such form.